config/opal_mca.m4

```diff
@@ -176,14 +179,16 @@ of type-component pairs.  For example, --enable-mca-no-build=pml-ob1])
         msg="none (dlopen disabled)"
     elif test -z "$enable_mca_dso"; then
         DSO_all=0
-        msg=default
+        DSO_accelerator=1
+        msg="accelerator framework only"
```

Comment:
Maybe do this:
```suggestion
        msg="accelerator framework only (default)"
```

Reply:
don't we have to add btl/smcuda and rcache/gpusm,, rcache/rgpusm also to the list of components that are compiled as dso's by default? Otherwise they will pull in the cuda libraries as well

Reply:
> don't we have to add btl/smcuda and rcache/gpusm,, rcache/rgpusm also to the list of components that are compiled as dso's by default? Otherwise they will pull in the cuda libraries as well

I'm okay with this although I thought we only wanted this DSO by default for accelerator framework.

Reply:
Lets discuss this in the meeting tomorrow. I think it probably comes down the precise goal that we want to achieve with making the accelerator components built as dsos

Reply:
hmm, i think something in the configury is broken.  I seem to only be able to set DSO builds at the framework level and not at the individual component level.  there are a number of differences between opal_mca.m4 in the main/5.0.0 and 4.1.x.  investigating...